src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -163,8 +163,11 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
             id="WI_details_reporter_img" />          
           <div class="pull-left marginR10">                          
-              <span id="WI_details_reporter_user"> 
-                {{workItem.relationalData?.creator?.attributes.fullName || 'No creator found'}} 
```

Comment:
@nimishamukherjee , @aslakknutsen 
Just wanted to get more clarification on fullName behavior ,couple of questions below
- Is core setting up fullName's value to Gtihub name in case of ,FullName = null ?
- Or there is a field in DB for Github username which shows up instead of Fullname ,when its null?